packages/api/lambdas/sf-scheduler.js

```diff
@@ -30,9 +35,10 @@ const getCollection = (collection) => {
  * @returns {Promise}
  */
 async function handleScheduleEvent(event) {
+  logger.debug(`Getting collection from API that matches ${JSON.stringify(event.collection)}`);
```

Comment:
Any reason not to put this log in `getApiCollection`? Just curious

Reply:
Excellent suggestion!  7a996f2df